Tests/UIKitTests/UIAlertControllerExtensionsTests.swift

```diff
@@ -14,86 +14,111 @@ final class UIAlertControllerExtensionsTests: XCTestCase {
         XCTAssertEqual(action?.style, .destructive)
         XCTAssertEqual(action?.isEnabled, false)
     }
-
+    
     func testSelector() {}
-
+    
     func testAddTextField() {
         let alertController = UIAlertController(title: "Title", message: "Message", preferredStyle: .alert)
-
+        
         let selector = #selector(testSelector)
-
+        
         alertController.addTextField(
             text: "TextField",
             placeholder: "PlaceHolder",
             editingChangedTarget: self,
             editingChangedSelector: selector)
-
+        
         XCTAssertEqual(alertController.textFields?.count, 1)
-
+        
         let textField = alertController.textFields?.first
-
+        
         XCTAssertEqual(textField?.text, "TextField")
         XCTAssertEqual(textField?.placeholder, "PlaceHolder")
         XCTAssertNotNil(textField?.allTargets)
         XCTAssertNotNil(textField?.actions(forTarget: self, forControlEvent: .editingChanged))
     }
-
+    
     func testMessageInit() {
         let alertController = UIAlertController(
             title: "Title",
             message: "Message",
             defaultActionButtonTitle: "Ok",
             tintColor: .blue)
-
+        
         XCTAssertNotNil(alertController)
-
+        
         XCTAssertEqual(alertController.title, "Title")
         XCTAssertEqual(alertController.message, "Message")
         XCTAssertEqual(alertController.view.tintColor, .blue)
-
+        
         XCTAssertEqual(alertController.actions.count, 1)
-
+        
         let defaultAction = alertController.actions.first
-
+        
         XCTAssertEqual(defaultAction?.title, "Ok")
         XCTAssertEqual(defaultAction?.style, .default)
     }
-
+    
     enum TestError: Error { case error }
-
+    
     func testErrorInit() {
         let error = TestError.error
-
+        
         let alertController = UIAlertController(
             title: "Title",
             error: error,
             defaultActionButtonTitle: "Ok",
             tintColor: .red)
-
+        
         XCTAssertNotNil(alertController)
-
+        
         XCTAssertEqual(alertController.title, "Title")
         XCTAssertEqual(alertController.message, error.localizedDescription)
         XCTAssertEqual(alertController.view.tintColor, .red)
-
+        
         XCTAssertEqual(alertController.actions.count, 1)
-
+        
         let defaultAction = alertController.actions.first
-
+        
         XCTAssertEqual(defaultAction?.title, "Ok")
         XCTAssertEqual(defaultAction?.style, .default)
     }
+    
+    func testLocalizeErrorAlert() {
+        
+        let alert = UIAlertController(localizedError: CustomError.netWorkError)
+        XCTAssertNotNil(alert)
+        
+        XCTAssertEqual(alert.title, "Wrong URL")
+        XCTAssertEqual(alert.message, "Make sure you don't make any typo")
+    }
+
+    
 }
 
+    enum CustomError: Error {
+        case netWorkError
+    }
```

Comment:
We could move all of this into the test case above so that it is more self contained

```swift
func testLocalizeErrorAlert() {
    enum CustomError: Error, LocalizedError {
        case networkError
        var errorDescription: String? {
            return "Wrong URL"
        }
        var recoverySuggestion: String? {
            return "Make sure you don't make any typo"
        }
    }
    let alert = UIAlertController(localizedError: CustomError.netWorkError)
    XCTAssertNotNil(alert)

    XCTAssertEqual(alert.title, "Wrong URL")
    XCTAssertEqual(alert.message, "Make sure you don't make any typo")
}
```